frontend/src/pages/ActivityReport/Pages/ReviewSubmit.js

```diff
@@ -1,24 +1,122 @@
-import { Button } from '@trussworks/react-uswds';
-
-const ReviewSubmit = ({ allComplete, onSubmit }) => (
-  <>
-    <Helmet>
-      <title>Review and submit</title>
-    </Helmet>
-    <div>
-      Review and submit
-      <br />
-      <Button disabled={!allComplete} onClick={onSubmit}>Submit</Button>
-    </div>
-  </>
-);
+
+import { fetchApprovers } from '../../../fetchers/activityReports';
+import MultiSelect from '../../../components/MultiSelect';
+import Container from '../../../components/Container';
+
+const defaultValues = {
+  approvingManagers: null,
+  additionalNotes: null,
+};
+
+const ReviewSubmit = ({
+  initialData, allComplete, onSubmit, submitted, reviewItems,
+}) => {
+  const [loading, updateLoading] = useState(true);
+  const [possibleApprovers, updatePossibleApprovers] = useState([]);
+
+  useEffect(() => {
+    updateLoading(true);
+    const fetch = async () => {
+      const approvers = await fetchApprovers();
+      updatePossibleApprovers(approvers);
+      updateLoading(false);
+    };
+    fetch();
+  }, []);
+
+  const {
+    handleSubmit, register, formState, control,
+  } = useForm({
+    mode: 'onChange',
+    defaultValues: { ...defaultValues, ...initialData },
+  });
+
+  const onFormSubmit = (data) => {
+    onSubmit(data);
+  };
+
+  const {
+    isValid,
+  } = formState;
+
+  const valid = allComplete && isValid;
+
+  return (
+    <>
+      <Helmet>
+        <title>Review and submit</title>
+      </Helmet>
+      <Accordion bordered={false} items={reviewItems} />
+      <Container skipTopPadding className="margin-top-2 padding-top-2">
+        <h3>Submit Report</h3>
+        {submitted
+        && (
+        <Alert noIcon className="margin-y-4" type="success">
+          <b>Success</b>
+          <br />
+          This report was successfully submitted for approval
+        </Alert>
+        )}
+        {!allComplete
+        && (
+        <Alert noIcon className="margin-y-4" type="error">
+          <b>Incomplete report</b>
+          <br />
+          This report cannot be submitted until all sections are complete
+        </Alert>
+        )}
+        <Form className="smart-hub--form-large" onSubmit={handleSubmit(onFormSubmit)}>
+          <Fieldset className="smart-hub--report-legend smart-hub--form-section" legend="Additional Notes">
+            <Label htmlFor="additionalNotes">Additional notes for this activity (optional)</Label>
+            <Textarea inputRef={register} id="additionalNotes" name="additionalNotes" />
+          </Fieldset>
+          <Fieldset className="smart-hub--report-legend smart-hub--form-section" legend="Review and submit report">
+            <p className="margin-top-4">
+              Submitting this form for approval means that you will no longer be in draft mode.
+              Please review all information in each section before submitting to your manager for
+              approval.
+            </p>
+            <MultiSelect
```

Comment:
I can't figure out why - but when I run this locally it has a weird default value on the approvers list.

<img width="535" alt="Screen Shot 2021-01-05 at 5 42 49 PM" src="https://user-images.githubusercontent.com/285842/103707884-7ce7bb80-4f7d-11eb-82cc-c11f39ba4d19.png">

Reply:
I left in some old data simulating loading managers/additional notes from an already saved report. I'll remove the old data.